jsonschema2pojo-core/src/test/java/org/jsonschema2pojo/util/LocalHttpServerBuilder.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright © 2010-2014 Nokia
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jsonschema2pojo.util;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import org.apache.commons.io.IOUtils;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.Executors;
+
+
+/**
+ * @author {@link "https://github.com/s13o" "s13o"}
+ * @since 3/16/2017
+ */
+public class LocalHttpServerBuilder {
```

Comment:
I know it doesn't seem like one endpoint is worth bringing in a library, but the project will have to lug this implementation around.  We should use some kind of library for this.  This entire PR should be somewhere in the 20 line range.